package.json

```diff
@@ -37,10 +38,10 @@
     "lab": "./bin/lab"
   },
   "scripts": {
-    "test": "./bin/_lab -fL -t 100 -m 3000",
+    "test": "node bin/_lab -fL -t 100 -m 3000",
     "posttest": "npm run lint-md",
-    "test-cov-html": "./bin/_lab -fL -r html -m 3000 -o coverage.html",
-    "lint": "./bin/_lab -d -f -L",
+    "test-cov-html": "node bin/_lab -fL -r html -m 3000 -o coverage.html",
+    "lint": "node bin/_lab -d -f -L",
```

Comment:
Unfortunately, Windows' cmd and Powershell doesn't understand the `./` syntax.
By using `node` invokation, we ensure a portable command.